.github/workflows/checks.yml

```diff
@@ -3,10 +3,10 @@ name: checks
 on:
   push:
     branches:
-      - master
+    - master
   pull_request:
     branches:
-      - master
+    - master
```

Comment:
Can we keep the style? If we want to change it, it can be done in a separate PR.

Reply:
Just called the YAML formatter on this file and this was collateral.

Reply:
I never `git add file`, I always use `git add -p`.

Given this is from the yaml formatter introduced here, it is okay to have it, but please do it for the whole codebase. And preferably, in a different commit.

Reply:
> I never `git add file`, I always use `git add -p`.

Not sure how this is relevant here, but 👍 for you. I'll remove this change, no worries. I'm not of fan of different commits, so my changes will always be one-commit 🤷 per feature/bugfix.